src/nekontrol/interactive/commands/__init__.py

```diff
@@ -0,0 +1,9 @@
+from . import run
+from . import submit
+from . import test
+
+__all__ = [
+    run.__name__,
+    submit.__name__,
+    test.__name__,
+]
```

Comment:
not sure what is going on in this file. it seems i have to "re-export". would appreciate a look here.

Reply:
if i remove the file's contents (or the entire file, same result) i get this error

```
  File "/home/gustav/comp/nekontrol/src/nekontrol/interactive/cli.py", line 49, in test
    commands.test.test(file_path, problem, config)
    ^^^^^^^^^^^^^
AttributeError: module 'nekontrol.interactive.commands' has no attribute 'test'
```

Reply:
Module != namespace, so if you want to use it like a namespace you have to reexport values, yes.

Reply:
Modules and namespaces being different explains a bit of the problems I have with Python.